exercises/practice/bank-account/.docs/instructions.append.md

```diff
@@ -14,6 +14,14 @@ For example: multiple goroutines may be depositing and withdrawing money
 simultaneously, two withdrawals occurring concurrently should not be able
 to bring the balance into the negative.
 
+To test that your code handles concurrency properly and does not introduce
```

Comment:
Since this content is visible for people working on the website as well as working locally, I would recommend to change it slightly so website users are not confused, e.g.
```suggestion
When working locally, you can test that your code handles concurrency properly and does not introduce
```